submissions/sayckl/friends-app/script.js

```diff
@@ -0,0 +1,115 @@
+const cards = document.querySelector('.cards');
+const az = document.querySelector('.a-z');
+const za = document.querySelector('.z-a');
+const min = document.querySelector('.min');
+const max = document.querySelector('.max');
+const male = document.querySelector('.male');
+const female = document.querySelector('.female');
+const all = document.querySelector('.all');
+const reset = document.querySelector('.reset');
+let staticPeople;
+let people;
+
+const menu = document.querySelector('.menu-btn');
+const aside = document.querySelector('aside');
+menu.addEventListener('click', () => {
+  menu.classList.toggle('menu-btn_active');
+  aside.classList.toggle('show_aside');
+});
+
+fetch("https://randomuser.me/api/?results=40&nat=us")
+  .then(function(response) {
+    if (response.ok) {
+      return response.json();
+    }
+  })
+  .then(function(data) {
+     staticPeople = data.results;
+     people = staticPeople;
+     add(staticPeople);
+   });
+
+function add(people){
+  people.forEach(function(elem) {
+    capitals(elem);
+    const div = document.createElement('div');
+    div.className = 'card';
+    div.innerHTML = `<img src="${elem.picture.large}">
+    <div class="name">${elem.name.first} ${elem.name.last}</div>
+    <div class="age">Age: ${elem.dob.age}</div>
+    <div class="location">Location: ${elem.location.city}</div>
+    <div class="phone">Phone: ${elem.phone}</div>`
+
+    cards.appendChild(div);
+  });
+};
+
+function capitals(elem) {
+  elem.name.first = capitalFirstLet(elem.name.first);
+  elem.name.last = capitalFirstLet(elem.name.last);
+  elem.location.city = capitalFirstLet(elem.location.city);
+}
+
+function capitalFirstLet(word) {
+  return word.charAt(0).toUpperCase() + word.slice(1);
+}
+
+az.addEventListener('click', () => {
```

Comment:
`az` is a bad naming - it is very unobvious. Please, find the right name

Reply:
changed to textASC
maybe better names/fullname ASC

Reply:
https://github.com/kottans/frontend-2019-p2p/pull/77#discussion_r296151214

and with `ascending`. Better use full words in names of variables